packages/api-server/src/filter-web3-tx.ts

```diff
@@ -0,0 +1,359 @@
+import { HexString } from "@ckb-lumos/base";
+import { Script } from "@ckb-lumos/base";
+import { Hash, HexNumber } from "@ckb-lumos/base";
+import {
+  HexU32,
+  U32,
+  L2Transaction,
+  L2TransactionReceipt,
+  GodwokenClient,
+  schemas,
+  U64,
+} from "@godwoken-web3/godwoken";
+import { Reader } from "ckb-js-toolkit";
+import { envConfig } from "./base/env-config";
+import { EthTransaction, EthTransactionReceipt } from "./base/types/api";
+import { Uint128, Uint32, Uint64 } from "./base/types/uint";
+import { PolyjuiceSystemLog, PolyjuiceUserLog } from "./base/types/gw-log";
+
+const CKB_SUDT_ACCOUNT_ID: U32 = 1;
+
+const EMPTY_ADDRESS = "0x" + "00".repeat(20);
+
+const GW_LOG_POLYJUICE_SYSTEM = "0x2";
```

Comment:
can we move these constant var to constant.ts file?

Reply:
updated.